layers/core_checks/cc_cmd_buffer_dynamic.cpp

```diff
@@ -110,7 +110,19 @@ bool CoreChecks::ValidateDrawDynamicState(const CMD_BUFFER_STATE &cb_state, cons
     skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_EXCLUSIVE_SCISSOR_ENABLE_NV_SET, cmd_type,
                                     vuid.dynamic_exclusive_scissor_enable_07879);
 
-    const auto rp_state = pipeline.RasterizationState();
+    // VK_EXT_extended_dynamic_state
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_CULL_MODE_SET, cmd_type, vuid.dynamic_cull_mode_07840);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_FRONT_FACE_SET, cmd_type, vuid.dynamic_front_face_07841);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_PRIMITIVE_TOPOLOGY_SET, cmd_type, vuid.dynamic_primitive_topology_07842);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_DEPTH_TEST_ENABLE_SET, cmd_type, vuid.dynamic_depth_test_enable_07843);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_DEPTH_WRITE_ENABLE_SET, cmd_type, vuid.dynamic_depth_write_enable_07844);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_DEPTH_COMPARE_OP_SET, cmd_type, vuid.dynamic_depth_compare_op_07845);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_DEPTH_BOUNDS_TEST_ENABLE_SET, cmd_type,
+                                    vuid.dynamic_depth_bound_test_enable_07846);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_STENCIL_TEST_ENABLE_SET, cmd_type, vuid.dynamic_stencil_test_enable_07847);
+    skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_STENCIL_OP_SET, cmd_type, vuid.dynamic_stencil_op_07848);
```

Comment:
IMO a little more clear.

```suggestion
    // VK_EXT_extended_dynamic_state
    {
        skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_CULL_MODE_SET, cmd_type, 
        ...
        skip |= ValidateCBDynamicStatus(cb_state, CB_DYNAMIC_STENCIL_OP_SET, cmd_type, vuid.dynamic_stencil_op_07848);
    }
```